Tests/L0/Maven/_suite.ts

```diff
@@ -214,23 +165,21 @@ describe('Maven Suite', function() {
         // Set up the task runner with the test settings
         var taskRunner:tr.TaskRunner = setupDefaultMavenTaskRunner();
         taskRunner.setInput('pmdAnalysisEnabled', 'true');
-        taskRunner.setInput('test.stagingDirectory', testStgDir);
+        taskRunner.setInput('test.artifactStagingDirectory', testStgDir);
         taskRunner.setInput('test.sourcesDirectory', testSrcDir);
 
         // Act
         taskRunner.run()
             .then(() => {
-                //console.log(taskRunner.stdout);
+                console.log(taskRunner.stdout);
 
                 // Assert
                 assert(taskRunner.resultWasSet, 'should have set a result');
                 assert(taskRunner.stdout.length > 0, 'should have written to stdout');
-                assert(taskRunner.succeeded, 'task should have succeeded');
+                assert(taskRunner.failed, 'task should have failed');
```

Comment:
This was a bit of an oversight, spotted and corrected once test setup was fixed.